packages/teams-js/test/private/copilot.spec.ts

```diff
@@ -319,6 +338,28 @@ describe('copilot', () => {
 
         await expect(promise).rejects.toThrowError('Error deserializing eligibility information');
       });
+
+      it('getEligibilityInfo should throw if AppEligibilityInformation.featureSet.serverFeatures or uxFeatures is undefined', async () => {
```

Comment:
Suggest adding two separate unit tests, one for `serverFeatures` and one for `uxFeatures` being undefined so it's clearer if there was a failure, one which caused it.